colossalai/inference/modeling/models/nopadding_baichuan.py

```diff
@@ -137,6 +136,7 @@ def __init__(
             self.alibi_slopes = get_alibi_slopes(config.num_attention_heads, device=attn_qproj_w.device)[
                 slopes_start : slopes_start + num_heads
             ].contiguous()
+            self.alibi_slopes = nn.Parameter(self.alibi_slopes)
```

Comment:
Why adding it as a param?

Reply:
If the input model is on the CPU, alibi_slopes will also be on the CPU. Later, the ModelWrapper will move the model to the GPU. However, since alibi_slopes is not a parameter, there will be an issue during execution due to the mismatch in devices.